package.json

```diff
@@ -5,6 +5,7 @@
   },
   "dependencies": {
     "@discordjs/rest": "^1.0.1",
+    "axios": "^1.6.2",
```

Comment:
**suggestion:** avoid using axios, let's use `fetch` which should be built-in javascript

Reply:
Why should we avoid using axios?